src/components/matrix.js

```diff
@@ -1,16 +1,22 @@
 import React from "react";
 
-import Code from "./code";
-
-export default class Commit extends React.Component {
-
+export default class GitHubSourceCode extends React.Component {
   getData() {
     const dataComponents = this.props.data.map(data => {
       return (
         <div key={data.id}>
-          <h4>{data.user}</h4>
-            <Code code={data.commit} />
+          <div className="matrix-information-section">
+            <h2>{data.project}</h2>
+            <h3>{data.githubUser}</h3>
+            <h4>{data.fileName}</h4>
+            <ul>{data.contributors.map(function(contributor) {
```

Comment:
ESLint is getting angry at this line, and that's why Travis is failing! Change it to `map(contributor => {...}` and it should be fine! 

Because it's a simple one line return you should even be able to do it like this: `map(contributor => <li key={contributor.id}>{contributor}</li>)`.